### ISSUE TITLE
Better description of application profiles, profiles in general and how they may be used

### ISSUE THREAD

**Issue by nicholascar** [w3c/dxwg#659]
From Irene Polikoff for the W3C SHACL CG:

> I found it hard to provide a meaningful review or answer questions about benefits without understanding what are the application profiles and profiles in general and how their descriptions may be used in practice.

**Comment by kcoyle** [comment]
I think in our minds we've thought that the guidance document will cover this, but we should make sure that all of the documents can be read and understood separately. This also may be related to some of the points in my [email](https://lists.w3.org/Archives/Public/public-dxwg-wg/2019Jan/0047.html) regarding definition of profile within the guidance document.

**Comment by aisaac** [comment]
@kcoyle I agree.
And your email is now included in the issue #537 which I'm hereby cross-referencing from this one...

**Comment by nicholascar** [comment]
This has been covered by recent (July) work on shared definitions. I will ensure that 3PWD of PROF is sent to the raiser (Irene) for review.

**Comment by kcoyle** [comment]
This is a public comment that needs a response and an agreement from the commenter. Do not close until that is complete.

**Comment by nicholascar** [comment]
Regarding definitions: Since the request for better descriptions of application profiles, profiles in general and how they may be used, we have updated both the definitions used in the profile documents and now the Profiles Vocabulary and the Content Negotiation by Profile document specifically define  profiles, application profiles, specifications and so on, see Section 3. In these Editor’s Drafts:

• https://w3c.github.io/dxwg/conneg-by-ap/#definitions – Content Negotiation by Profile
• https://raw.githack.com/w3c/dxwg/prof-3PWD-candidate/prof/#definitions – The Profiles Vocabulary

Regarding “how their descriptions may be used in practice”: in both of the documents linked to above, we have provided multiple examples of either the vocabulary or content negotiation Internet requests.

These comments have been forwarded to the Issue raiser, Irene Polikoff, via the DXWG public mailing list: https://lists.w3.org/Archives/Public/public-dxwg-comments/2019Sep/0000.html.
